.github/workflows/harvey.yml

```diff
@@ -16,6 +16,8 @@ jobs:
         sudo apt-get update
         sudo apt-get install -y rc qemu go-dep qemu-utils expect qemu-system-x86
         go get harvey-os.org/cmd/ufs
+    - name: install source we build into the image
+        git clone --depth 1 --branch go1.16.6 git@github.com:golang/go.git
```

Comment:
Should this be under a ‘run’ command like the others?

Reply:
Thanks, let's see what that does.